### PR TITLE
[11.0] Print fiscal code in invoice

### PR DESCRIPTION
**Pull request by joezsweet**

### CODE REVIEW

**Comment by eLBati** [comment]
Hi @joezsweet thanks for your contribution.

You should rather port to 11 [l10n_it_fiscalcode_invoice](https://github.com/OCA/l10n-italy/tree/10.0/l10n_it_fiscalcode_invoice) module.

Here is the guide https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0 if needed

**Comment by joezsweet** [comment]
What's the goal to have a module just to print fiscalcode in invoice?

**Comment by eLBati** [comment]
In this case , I don't see reasons to have a separate module indeed.
Reopening it

File: l10n_it_fiscalcode/view/report_invoice_document.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+	<template id="print_invoice_inherit" inherit_id="account.report_invoice_document">
+		<xpath expr="//div[@name='invoice_address']" position="after">
```
Comment (eLBati) [inline l10n_it_fiscalcode/view/report_invoice_document.xml]: @joezsweet why not inside `invoice_address` div, along with other customer data? You would probably save space

**Comment by eLBati** [comment]
@joezsweet thanks.

Could you squash the 3 commits in 1?

Assuming `origin` is `https://github.com/OCA/l10n-italy.git` and is up to date (`git fetch origin`), run `git rebase -i origin/11.0`. Then squash the last 2 commits in the first one. Then `git push -f`

**Comment by eLBati** [comment]
@joezsweet it seems you created commits directly from github.com
If you prefer, I can squash the commits for you

**Comment by joezsweet** [comment]
Yeah, i've done directly in github because of few and fast changes; feel free to squash commits. Probably is better to use `Fiscal code:` as label for translation compatibility. Could u change it?

**Comment by eLBati** [comment]
Done

**Comment by tafaRU** [comment]
Shouldn't be better instead migrate https://github.com/OCA/l10n-italy/tree/10.0/l10n_it_fiscalcode_invoice ?

**Comment by eLBati** [comment]
@tafaRU https://github.com/OCA/l10n-italy/pull/623#issuecomment-433606805

**Comment by joezsweet** [comment]
@tafaRU what should be the goal to have it in another module? just to print the fiscal code in invoice?

**Comment by eLBati** [comment]
Also, as `l10n_it_fiscalcode_invoice` has `'auto_install': True` and depends on `"l10n_it_fiscalcode", "account"`, and `l10n_it_fiscalcode` depends, implicitly, on `account`, `l10n_it_fiscalcode_invoice` would be always installed along with `l10n_it_fiscalcode`. So, there is no reason to have a separate module

**Comment by tafaRU** [comment]
ok, for consistency, I'd also apply the same changes to branch `10.0` then.

**Comment by eLBati** [comment]
@tafaRU I would not remove a module from a stable branch

**Pull request merged**